.circleci/config.yml

```diff
@@ -102,6 +102,9 @@ jobs:
                   command: |
                       diff $ABI_ORIGIN $ABI_COPY
                       rm $ABI_COPY
+            - run:
```

Comment:
I don't think we should run this in CI on every build. It's adding ~2.5 minutes to the ci time